sdk/trace/src/test/java/io/opentelemetry/sdk/trace/SdkTracerProviderTest.java

```diff
@@ -42,15 +44,44 @@ void setUp() {
   }
 
   @Test
-  void builder_HappyPath() {
-    assertThat(
-            SdkTracerProvider.builder()
-                .setClock(mock(Clock.class))
-                .setResource(mock(Resource.class))
-                .setIdGenerator(mock(IdGenerator.class))
-                .setTraceConfig(mock(TraceConfig.class))
-                .build())
-        .isNotNull();
+  void builder_fallbackServiceName() {
+    Resource resource = mock(Resource.class);
```

Comment:
Do we need this mock?

Reply:
probably not. Can just use `Resource.getEmpty()`, for sure.